x-pack/plugins/stack_alerts/public/alert_types/es_query/rule_common_expressions/rule_common_expressions.tsx

```diff
@@ -123,7 +135,19 @@ export const RuleCommonExpressions: React.FC<RuleCommonExpressionsProps> = ({
         popupPosition="upLeft"
         onChangeSelectedValue={onChangeSizeValue}
       />
-      <EuiSpacer size="s" />
+      <EuiSpacer size="m" />
+      <EuiFormRow>
+        <EuiCheckbox
+          data-test-subj="excludeHitsFromPreviousRunExpression"
+          checked={excludeHitsFromPreviousRun}
+          id="generated_42069051-1983-11ed-abf3-23049257d20d"
+          onChange={(event) => {
+            onChangeExcludeHitsFromPreviousRun(event.target.checked);
+          }}
+          label="Exclude hits from previous rule runs"
```

Comment:
Yes i forgot i18n part, fixing it.
We discussed this with Shani and Mike and thought that "duplicate" is a bit misleading. Because usually it is used for a single set of data but here we have separate sets. I think Lisa may help us on this :)

Reply:
I have put some alternatives for consideration in a separate document (e.g. "Deduplicate document matches") but since I need to catch up on the previous concerns about "duplicate" if you want to merge this as-is I can alter in a subsequent PR if necessary.